packages/website/src/SideNavigation.tsx

```diff
@@ -20,7 +20,7 @@ const NavLink: React.FunctionComponent<NavLinkProps> = ({href = '', label, disab
                 </div>
             ) : (
                 <Link href={href} prefetch={false}>
-                    <a className="navigation-menu-section-item-link">
+                    <a href={href} className="navigation-menu-section-item-link">
```

Comment:
Are you sure about that one? The parent Link works on master, no?

Reply:
It's to fix thoses errors in the console :disappointed: 
![image](https://user-images.githubusercontent.com/63734941/157760395-0238cfa3-32fd-4572-9142-c6a4d586bea8.png)